SwinUNETR/Pretrain/README.md

```diff
@@ -0,0 +1,74 @@
+# Model Overview
+![image](./assets/ssl_swin.png)
+
+This repository contains the code for self-supervised pre-training of Swin UNETR model[1,2] for medical image segmentation. Swin UNETR is the state-of-the-art on Medical Segmentation
+Decathlon (MSD) and Beyond the Cranial Vault (BTCV) Segmentation Challenge dataset.
+
+### Installing Dependencies
+Dependencies can be installed using:
+``` bash
+pip install -r requirements.txt
+```
+
+### Pre-trained Models
+
+We provide the self-supervised pre-trained weights for Swin UNETR backbone (CVPR paper [1]) in this <a href="https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/model_swinvit.pt"> link</a>.
+In the following, we describe steps for pre-training the model from scratch.
+
+## Datasets
+
+The following datasets were used for pre-training (~5050 3D CT images):
+
+- Head & Neck Squamous Cell Carcinoma (HNSCC) ([Link](https://wiki.cancerimagingarchive.net/display/Public/HNSCC))
+- Lung Nodule Analysis 2016 (LUNA 16) ([Link](https://luna16.grand-challenge.org/Data/))
+- TCIA CT Colonography Trial ([Link](https://wiki.cancerimagingarchive.net/display/Public/CT+COLONOGRAPHY/))
+- TCIA Covid 19 ([Link](https://wiki.cancerimagingarchive.net/display/Public/CT+Images+in+COVID-19/))
+- TCIA LIDC ([Link](https://wiki.cancerimagingarchive.net/display/Public/LIDC-IDRI/))
+
+Please see the corresponding the json files of each dataset in ```jsons``` folder.
```

Comment:
I uploaded the jsons to https://github.com/Project-MONAI/MONAI-extra-test-data/releases/tag/0.8.1,

please use links such as https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/dataset_HNSCC_0.json  and remove the json files from this PR.

Reply:
@wyli Thanks. Addressed.

Reply:
could you please also remove the content in the `json` folder? in general the data are not archived in this repo.

Reply:
sounds good